README.md

```diff
@@ -1,339 +1,31 @@
-a format that Dockstore supports and leverage that tool.
+These pipelines were developed by the [Washington University Division of Oncology](https://oncology.wustl.edu/), in collaboration with the [McDonnell Genome Institute](https://genome.wustl.edu/) and other members of the [Washington University School of Medicine](https://medicine.wustl.edu/).  Much of the code and structure is adapted from a sister repository of [workflows in the CWL language](https://github.com/genome/analysis-workflows/)
 
 
-# Common Errors
+## Getting Started
 
-## Out of Space
+### Workflows
+Download our repository with `git clone https://github.com/wustl-oncology/analysis-wdls.git`
 
-These errors indicate that the disk storage space has been
-filled. That part is pretty straightforward. The part that's a bit
-more of a pitfall is that _depending on what output_ this happened on,
-you need to change different disk sizes.
+Terra hosts an excellent guide to [getting started with WDL](https://support.terra.bio/hc/en-us/articles/360037117492-Overview-Getting-started-with-WDL) 
 
-If the failed write happened at `/cromwell_root` path, then `disks:
-"local-disk ..."` needs to be increased. However, if the failed write
-happens during to `stdin` or `stdout`, or any of the other standard
-Linux-y places, then you'll need to increase the value of
-`bootDiskSizeGb`. Cromwell in GCP mounts two disks, at minimum: the
-boot disk, and a local-disk. Boot disk handles all the operating
-system files, but local-disk is where almost all of your "work" is
-going to happen, besides piping between commands.
+### Workflow Execution
+These are primarily designed to be run with the cromwell workflow system, either on a self-hosted server through the Google Cloud Platform ([details here](https://github.com/wustl-oncology/cloud-workflows)), through [Terra](https://terra.bio/)) or on local compute clusters. They have been extensively used in these contexts, though the portability of WDL means that running on other platforms should be straightforward as well.
 
-## File missing
+### Docker
+In order to provide a portable environment, each tool in our workflow has a designated Docker container. [Download Docker here](https://www.docker.com/products/docker-desktop).
 
-This applies more to newly converted files than hardened ones _but_
-many runs failed because a file wasn't included in the
-instance. Generally, this happens because the CWL did not specify a
-secondaryFile that it assumed would exist next to the passed in
-file. This works on the cluster, because the tools just look for the
-file and it already sits where it's expected. This does not work on
-the cloud because that file is never sent to the instance. The
-solution is to add this parameter explicitly to the WDL and pass it
-through, top down.
+### Data
+Input "bundles" containing reference genomes and annotations are available for use/download along with documentation on how they were created.
```

Comment:
This one really cries out for a link 🙂

Reply:
Yeah, it's going to get one - on my to-do list for the next couple of days!